### PR TITLE
first version of the flow article

### PR DESCRIPTION
**Pull request by frontendphil**
I wrote this down in one go so there might be some weird sentences in there.

### CODE REVIEW

File: _posts/2018-07-10-integrating-flow.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+The actual type check didn't work for our large code base for various reasons.
+We decided to give flow a go nonetheless hoping that the ecosystem would evolve eventually.
+
+**TL;DR** we started adding types all over the place without ever checking them.
+
+## Pulling the trigger
+
+A couple of weeks ago I was once again looking at an error that could have been prevented by having proper type checks.
+It was that moment, when I realized that we should finally start enforcing the code to type check on each PR.
+But since we never did this before, but had added the `// @flow` pragma to over 800 files, we were left with a couple thousand violations.
```
Comment (steffektif) [inline _posts/2018-07-10-integrating-flow.md]: There is quite an obvious typo there. I think the second `but` wants to be a ´we´ or something similar.

**Comment by hilton** [comment]
Nicely written story - I wouldn't change much, although Stef has a good that you would enlarge the audience if you added a _Type checking_ heading at the top, with a single paragraph that uses 2-3 sentences to summarise what the topic is, maybe with links to https://en.wikipedia.org/wiki/Type_system#Type_checking and https://en.wikipedia.org/wiki/Flow-sensitive_typing for readers who want to catch up.

Fixing a few cosmetic issues would improve readability:

* Correctly capitalise product names, e.g. Flow, and link to their web sites, rather than writing ‘flow’ or ‘`flow`’.
* Don’t use abbreviations; change PR to pull request.
* Use single ‘curly’ quotes instead of straight or double quotes
* Write numbers below 21 as words, e.g. ‘two’ instead of ‘2’

**Comment by hilton** [comment]
@frontendphil I added a photo and changed the title to fit better on the front page and mention JavaScript - hope that’s okay

**Comment by frontendphil** [comment]
@hilton @steffektif thanks for your remarks!

**Comment by frontendphil** [comment]
@steffektif @hilton 🏓

**Comment by hilton** [comment]
On balance, I think it’s better to only link to a product the first time it’s mentioned. Sorry!

I also added a link to the one product that was missing one :)

**Pull request merged**